src/plugins/credential_management/public/components/create_credential_wizard/components/header/header.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+
+import { EuiBetaBadge, EuiSpacer, EuiTitle, EuiText, EuiCode, EuiLink } from '@elastic/eui';
+
+import { i18n } from '@osd/i18n';
+import { FormattedMessage } from '@osd/i18n/react';
+import { DocLinksStart } from 'opensearch-dashboards/public';
+import { useOpenSearchDashboards } from '../../../../../../opensearch_dashboards_react/public';
+import { CredentialManagementContext } from '../../../../types';
+export const Header = ({
+  isBeta = true,
```

Comment:
is there a use case we want to set this `isBeta` to false? when this feature go GA, we will remove all beta related code, so such switch seems not necessary

Reply:
is experimental GA same as Beta?

Reply:
we are going to release it as experimental, thus we want to let user know this is experimental. I assume you are showing this as `beta` for this purpose. my point is that, we don't need a switch here since we won't change the `isBeta` value in any case

Reply:
removed the switch